WalletWasabi.Tests/UnitTests/AllFeeEstimateTests.cs

```diff
@@ -263,11 +266,12 @@ target switch
 			};
 
 		var allFee = await mockRpc.EstimateAllFeeAsync();
-		Assert.Equal(3_000, allFee.Estimations[2]);
-		Assert.True(allFee.Estimations[3] > 500);
+		Assert.Equal(140, allFee.Estimations[2]);
+		Assert.Equal(124, allFee.Estimations[144]);
 		Assert.True(allFee.Estimations[1008] > 1);
 	}
 
+
```

Comment:
```suggestion
```

Reply:
Now we are only two. There is not more place for this kind of comments/suggestions.